<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1482</width>
    <height>949</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MplWidget" name="MplWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1261</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_1D_data">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>0</y>
      <width>221</width>
      <height>213</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_1D_Data">
       <property name="text">
        <string>Data : </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_1D_DataName">
       <property name="text">
        <string>No Signal Selected</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_1D_Type">
       <property name="text">
        <string>Wavelet</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_1D_Type"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_1D_Level">
       <property name="text">
        <string>Level</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_1D_Level"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_1D_staticstics">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>210</y>
      <width>221</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox_1D_StandartDeviation">
       <property name="text">
        <string>Standart Deviation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_1D_Min">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox_1D_Average">
       <property name="text">
        <string>Average</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBox_1D_Max">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox_1D_Entropy">
       <property name="text">
        <string>Entropy</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBox_1D_Kurtosis">
       <property name="text">
        <string>Kurtosis</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBox_1D_Median">
       <property name="text">
        <string>Median</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBox_1D_Skewness">
       <property name="text">
        <string>Skewness</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_1D_Analyze">
       <property name="text">
        <string>Analyze</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_save_to_db">
       <property name="text">
        <string>Save To Database</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget_1D_matrix">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>660</y>
      <width>1481</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1482</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLoad_2">
     <property name="title">
      <string>Load</string>
     </property>
     <addaction name="actionImage"/>
     <addaction name="actionFolder"/>
    </widget>
    <addaction name="menuLoad_2"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuLoad"/>
  </widget>
  <action name="actionImage">
   <property name="text">
    <string>Signal</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionFolder">
   <property name="text">
    <string>Folder</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
